CBLClient/Replication.py

```diff
@@ -29,14 +29,16 @@ def configure(self, source_db, target_url=None, target_db=None,
                   replication_type="push_pull", continuous=False,
                   push_filter=False, pull_filter=False, channels=None,
                   documentIDs=None, replicator_authenticator=None,
-                  headers=None, filter_callback_func='', conflict_resolver=""):
+                  headers=None, filter_callback_func='', conflict_resolver="", heartbeat=""):
```

Comment:
could you please add single cotes for all

Reply:
updated in a new push